llvm/lib/Target/AArch64/AArch64TargetTransformInfo.cpp

```diff
@@ -3259,8 +3265,12 @@ InstructionCost AArch64TTIImpl::getMemoryOpCost(unsigned Opcode, Type *Ty,
   // of <vscale x 1 x eltty> yet, so return an invalid cost to avoid selecting
   // it. This change will be removed when code-generation for these types is
   // sufficiently reliable.
+  // We also cannot handle loads or stores involving scalable vectors of i1.
```

Comment:
Perhaps "We also only support full register predicate loads and stores."?

Reply:
Done!